### PR TITLE
[[ Bug 19400 ]] Unmangle java module docs

### PR DESCRIPTION
**Pull request by livecodeali**

### CODE REVIEW

File: libscript/src/java.lcb [hunk 1]
```diff
@@ -26,8 +26,8 @@ Description:
 > Java installation (usually at 
 > `/Library/Java/JavaVirtualMachines/jdk1.7.0_55.jdk/Contents/Home`). 
 > On Linux, your `LD_LIBRARY_PATH` must be set to the folder containing 
-> the `libjvm.so` library (usually at ${JAVA_HOME}/jre/lib/amd64/server)
-> on 64-bit Linux.
+> the `libjvm.so` library (usually at ${JAVA_HOME}/jre/lib/amd64/server
+> on 64-bit Linux).
```
Comment (peter-b) [inline libscript/src/java.lcb]: It might be a good idea to put the path in `code markup`.

**Comment by peter-b** [comment]
@livecode-vulcan review ok 1f3d5c1

**Comment by livecode-vulcan** [comment]
💙 review by @peter-b ok 1f3d5c14491455114d4e0347f3ad5f9ac83906e4

**Comment by livecode-vulcan** [comment]
😞 test failure 1f3d5c14491455114d4e0347f3ad5f9ac83906e4

* try-community-linux-x86: [success](https://vulcan.livecode.com/builders/try-community-linux-x86/builds/1054)
* try-community-mac: [success](https://vulcan.livecode.com/builders/try-community-mac/builds/1057)
* try-community-linux-x86_64: [success](https://vulcan.livecode.com/builders/try-community-linux-x86_64/builds/1049)
* try-community-emscripten: [success](https://vulcan.livecode.com/builders/try-community-emscripten/builds/1055)
* try-community-ios: [success](https://vulcan.livecode.com/builders/try-community-ios/builds/1059)
* try-community-win-x86: [failure](https://vulcan.livecode.com/builders/try-community-win-x86/builds/1071)
* try-community-android-armv6: [success](https://vulcan.livecode.com/builders/try-community-android-armv6/builds/1049)

**Comment by livecodeali** [comment]
@peter-b Done, since the build failed anyway!

**Comment by peter-b** [comment]
@livecode-vulcan review ok 37bde6f

**Comment by livecode-vulcan** [comment]
💙 review by @peter-b ok 37bde6feb5ba44f217b8fd64e86faaa4d669568c

**Comment by livecode-vulcan** [comment]
😞 test failure 37bde6feb5ba44f217b8fd64e86faaa4d669568c

* try-community-linux-x86: [pending](https://vulcan.livecode.com/builders/try-community-linux-x86/builds/1056)
* try-community-mac: [success](https://vulcan.livecode.com/builders/try-community-mac/builds/1059)
* try-community-linux-x86_64: [pending](https://vulcan.livecode.com/builders/try-community-linux-x86_64/builds/1051)
* try-community-emscripten: [pending](https://vulcan.livecode.com/builders/try-community-emscripten/builds/1057)
* try-community-ios: [pending](https://vulcan.livecode.com/builders/try-community-ios/builds/1061)
* try-community-win-x86: [failure](https://vulcan.livecode.com/builders/try-community-win-x86/builds/1073)
* try-community-android-armv6: [success](https://vulcan.livecode.com/builders/try-community-android-armv6/builds/1051)

**Comment by peter-b** [comment]
@livecode-vulcan retry 37bde6f

**Comment by livecode-vulcan** [comment]
😎 test success 37bde6feb5ba44f217b8fd64e86faaa4d669568c

* try-community-linux-x86: [success](https://vulcan.livecode.com/builders/try-community-linux-x86/builds/1058)
* try-community-mac: [success](https://vulcan.livecode.com/builders/try-community-mac/builds/1061)
* try-community-linux-x86_64: [success](https://vulcan.livecode.com/builders/try-community-linux-x86_64/builds/1053)
* try-community-emscripten: [success](https://vulcan.livecode.com/builders/try-community-emscripten/builds/1059)
* try-community-ios: [success](https://vulcan.livecode.com/builders/try-community-ios/builds/1063)
* try-community-win-x86: [success](https://vulcan.livecode.com/builders/try-community-win-x86/builds/1075)
* try-community-android-armv6: [success](https://vulcan.livecode.com/builders/try-community-android-armv6/builds/1053)

**Pull request merged**